diff -urN hedgewars-src-0.9.19.1-orig/CMakeLists.txt hedgewars-src-0.9.19.1-patched/CMakeLists.txt
--- hedgewars-src-0.9.19.1-orig/CMakeLists.txt	2013-05-31 07:11:20.000000000 +1100
+++ hedgewars-src-0.9.19.1-patched/CMakeLists.txt	2013-06-02 02:13:03.000000000 +1100
@@ -346,7 +346,7 @@
     #-XLA is a beta fpc flag that renames libraries before passing them to the linker
     #we also have to pass PHYSFS_INTERNAL to satisfy windows runtime requirements
     #(should be harmless on other platforms)
-    list(APPEND pascal_flags "-XLAphysfs=${physfs_output_name}" "-dPHYSFS_INTERNAL")
+    list(APPEND pascal_flags "-XLAphysfs=${physfs_output_name} -lgcc_s" "-dPHYSFS_INTERNAL")
 endif()
 
 find_package_or_disable_msg(FFMPEG NOVIDEOREC "Video recording will not be built")
@@ -376,6 +376,5 @@
     add_subdirectory(tools)
 endif()
 
-
-include(${CMAKE_MODULE_PATH}/CPackConfig.cmake)
+include(${CMAKE_MODULE_PATH}/cpackvars.cmake)
 
